### ISSUE TITLE
AZ-100T03A-ENU-LabGuide typo

### ISSUE THREAD

**Issue by simplescripter** [MicrosoftLearning/AZ-100-MicrosoftAzureInfrastructureDeployment#9]
Exercise 2, Task 3, Step 11 reads:  

"On the az1000302-vm0 - Networking blade, review the inbound port rules of the network security group assigned to the network interface of az1000301-vm0 to determine why your SSH connection via the private IP address was successful."

Both VM references should be to az1000302-vm0.  The second reference in the current instruction is to az1000301-vm0

**Comment by renaudwangler** [comment]
I totally agree with this remark which is, a month later, still uncorrected....
